charts/spire/charts/spire-agent/values.yaml

```diff
@@ -94,6 +94,22 @@ waitForIt:
     tag: latest-20230517
   resources: {}
 
+bash:
```

Comment:
Maybe a small comment here that this will be used to patch up the container to run as non-root?

Reply:
I was a bit on the fence if it should just be bash, or image stuck under fsGroupFix. Would the latter be more understandable? If we ever needed another image though, there may be some duplication.

Reply:
I think so yea.

Reply:
Updated